CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing to Distributed Ledger
+
+This document covers how to report issues and contribute code.
+
+## Topics
+
+* [Reporting Issues](#reporting-issues)
+* [Contributing Code](#contributing-code)
+
+# Reporting Issues
+
+This is a great way to contribute. Before reporting an issue, please review
+current open issues to see if there are any matches.
+
+## How to Report an Issue
+
+Issues should be reported to this project's Github Issues page: [https://github.com/hyperledger/sawtooth-hyper-directory/issues](https://github.com/hyperledger/sawtooth-hyper-directory/issues)
+
+When reporting an issue, please provide as much detail as possible about how
+to reproduce it. If possible, provide instructions for how to reproduce the
+issue within the vagrant development environment.
+
+Details are key. Please include the following:
+
+OS version
+Distributed Ledger version
+Environment details (virtual, physical, etc.)
+Steps to reproduce
+Actual results
+Expected results
+
+
+# Contributing Code
+
+Distributed Ledger is Apache 2.0 licensed and accepts contributions via
+[GitHub](https://github.com/hyperledger/) pull requests.
+
+## Ways to Contribute
+
+Contributions by the community help grow and optimize the capabilities of
+Hyperledger Sawtooth, and are the most effective method of having a positive
+impact on the project.
+
+## Different ways you can contribute
+
+Bugs or Issues (issues or defects found when working with
+    Sawtooth Hyper Directory)
+Core Features & Enhancements (expanded capabilities or optimization)
+New or Enhanced Documentation (improve existing documentation or create new)
+Testing Events and Results (functional, performance or scalability)
+
+## Commit Process
+
+Hyperledger Sawtooth Hyper Directory is Apache 2.0 licensed and accepts
+contributions via GitHub pull requests. When contributing code please do the
+following:
+
+
+
+Fork the repository and make your changes in a feature branch.
+Please include unit and integration tests for any new features and updates to
+existing tests.
+Please ensure the unit and integration tests run successfully. Both are run
+with ./bin/run_tests, but integration tests are only run if the environment
```

Comment:
bin/run_tests isn't in this repo. I do see a run_tests_ubuntu but no integration flag.